pkg/kubernetes/schema/schema.json

```diff
@@ -47,7 +47,7 @@
           },
           "additionalProperties": false,
           "required": [
-            "description", "manifests"
+            "manifests"
```

Comment:
description is no longer required, this updates the mixin schema to match the change that is already in porter